xbmc/interfaces/json-rpc/AudioLibrary.cpp

```diff
@@ -202,6 +202,9 @@ JSONRPC_STATUS CAudioLibrary::GetAlbums(const std::string &method, ITransportLay
     musicUrl.AddOption("genreid", static_cast<int>(filter["genreid"].asInteger()));
   else if (filter.isMember("genre"))
     musicUrl.AddOption("genre", filter["genre"].asString());
+  else if (filter.isMember("boxset"))
+    musicUrl.AddOption("boxset", filter["boxset"].asBoolean());
```

Comment:
I don't think boxed should be this kind of filter - see how rule filters happen by magic